js/components/controlBar.js

```diff
@@ -22,7 +22,13 @@ var ControlBar = React.createClass({
     this.isMobile = this.props.controller.state.isMobile;
     this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
     this.moreOptionsItems = null;
-    return {};
+    this.vr = this.props.controller
+      && this.props.controller.getVrParams
+      && this.props.controller.getVrParams();
+    
+    return {
+      currentVolumeHead: 0
```

Comment:
Should we actually be returning this since we are not explicitly setting the volume in the for this function?  is there a 'this' variable we can stick it into instead.